gridcomps/ExtData/ExtDataGridCompMod.F90

```diff
@@ -2113,8 +2113,10 @@ subroutine GetLevs(item, time, state, allowExtrap, rc)
         if (found) then
            call MakeMetadata(file,item%pfioCollection_id,metadata,__RC__)
         else
-           if (allowExtrap .and. (item%cyclic == 'n') ) then
+           if (index(item%file,'%') == 0) then
+              _ASSERT(.false.,'File ' // trim(item%file) // ' not found')
```

Comment:
I don't know what this does (@bena-nasa would know), but I wonder if we could make it a one-liner:
```fortran
_ASSERT(index(item%file,'%') /= 0, 'File ' // trim(item%file) // ' not found')
```

Probably got the logic wrong though. @tclune might know better.

Also don't know how to test it, so I could break MAPL. 😄

Reply:
That should work too. The test would be to put a typo in the filename in ExtData.rc of an input file with non-zero frequency and no time tokens in its name. We have a case of this in GCHP I can test with but it would be in an older MAPL and not testing GEOS.

Reply:
@mathomp4 - I prefer your version for clarity. But it is equivalent.

Side note:  any `_ASSERT(.false.,<msg>)`  should generally be written as `_FAIL(<msg>)`.     This should only arise when a failing conditional requires a few other actions prior to throwing the exception.

Reply:
Huh. I didn't know we had a `_FAIL()`. Good to know! 

I'll work with @bena-nasa to make sure I know how to trigger this. ExtData is still a bit of a black box to me :)